src/trinary_sse42.h

```diff
@@ -72,4 +73,161 @@ static inline bool validateTrytes_sse42(Trobject_t *trytes)
     return true;
 }
 
+static inline Trobject_t *trytes_from_trits_sse42(Trobject_t *trits)
+{
+    Trobject_t *trytes = NULL;
+    int8_t *src = (int8_t *) malloc(trits->len / 3);
+
+    const int block_8bit = BLOCK_8BIT(__m128i);
+    int8_t setMSB = 0x80;
+    __m128i tryteAlphabet[2] = {
+        _mm_setr_epi8('9', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J',
+                      'K', 'L', 'M', 'N', 'O'),
+        _mm_setr_epi8('P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', 0,
+                      0, 0, 0, 0)};
+    /* For shuffling the bytes of the input trits */
```

Comment:
I wish there are some macros for shuffling, otherwise it is almost impossible to maintain.

Reply:
I still have no idea about the better way to introduce macros for shuffling.
So I make the code remain the same.

Here comes two ways to introduce macros:
1. With many operations of shifting, masking and setting value.
However, it would be too tedious to give value for a `__m128i` array.
The performance would be affected, too.
2. Another way with macros:
```
     const __m128i shuffleLow[3] = {
-        _mm_setr_epi8(0x00, 0x03, 0x06, 0x09, 0x0C, 0x0F, setMSB, setMSB,
-                      setMSB, setMSB, setMSB, setMSB, setMSB, setMSB, setMSB,
-                      setMSB),
-        _mm_setr_epi8(setMSB, setMSB, setMSB, setMSB, setMSB, setMSB, 0x02,
-                      0x05, 0x08, 0x0B, 0x0E, setMSB, setMSB, setMSB, setMSB,
-                      setMSB),
-        _mm_setr_epi8(setMSB, setMSB, setMSB, setMSB, setMSB, setMSB, setMSB,
-                      setMSB, setMSB, setMSB, setMSB, 0x01, 0x04, 0x07, 0x0A,
-                      0x0D)};
+        _mm_setr_epi8(REPEAT(0, setMSB) COMMA(0) INDEX_3DIFF_0F COMMA(1)
+                          REPEAT(10, setMSB)),
+        _mm_setr_epi8(REPEAT(6, setMSB) COMMA(1) INDEX_3DIFF_2E COMMA(1)
+                          REPEAT(5, setMSB)),
+        _mm_setr_epi8(REPEAT(11, setMSB) COMMA(1) INDEX_3DIFF_1D COMMA(0)
+                          REPEAT(0, setMSB))};

```
I am not sure that this is acceptable or not.
It seems not helping much but it looks more elegant.

Reply:
The later looks better to me.

Reply:
They are all replaced with the latter format.